_includes/markdown/Markup.md

```diff
@@ -285,10 +285,96 @@ Combining automated and manual testing practices allows 10up to maintain a high
+
+### Best Practices &amp; Common Usage
+
+#### SVG icon in a button with text
+
+```
+<button type="button">
+    Menu
+    <svg viewBox="0 0 10 10"
+        class="svg-icon"
+        aria-hidden="true"
+        focusable="false">
+        <path d="m1 7h8v2h-8zm0-3h8v2h-8zm0-3h8v2h-8z"/>
+    </svg>
+</button>
+```
+
+Since the SVG is merely decorative and the "Menu" textually explains the button's purpose. Then we can leverage `aria-hidden="true"` and `focusable="false"` to hide the SVG-as-decorative-element from the accessibility tree.
+
+#### SVG icon in a button with _hidden_ text
+
+```
+<button type="button">
+    <svg class="svg-icon"
+        height="10"
+        width="10"
+        viewBox="0 0 10 10"
+        aria-hidden="true"
+        focusable="false">
+        <path d="m1 7h8v2h-8zm0-3h8v2h-8zm0-3h8v2h-8z"/>
+    </svg>
+    <span class="visually-hidden">
+        Menu
+    </span>
+</button>
+```
+
+The `<span>Menu</span>` still provides assistive technology with a textual explanation, while hiding the text from the viewer.
+
+The corresponding CSS for this:
+
+```
+.visually-hidden {
+    clip: rect(0, 0, 0, 0); // Clip the box to zero pixels.
+    height: 1px;
+    overflow: hidden; // Hide any overflowing elements or text.
+    position: absolute; // Move the text out of the flow of the container.
+    white-space: nowrap; // Text won't wrap to a second line.
+    width: 1px; // Reduce its height and width to just one pixel.
+}
+```
+
+#### SVG as non-decorative element
+
+If you would like to use an inline SVG. Here are some considerations to improve experience for assistive technology:
+
+* Use a descriptive `<title>` within the SVG.
+* Assign `role="image"`
+* Use `aria-labeledby` to help call out the title and description.
+
+Let's put it all together in an example:
+
+```
+<svg version="1.1" width="50" height="50" role="img" aria-labelledby="title-2 desc-2">
+    <title id="title-2">red square</title>
+    <desc id="desc-2">A plain red square defined at 50 pixels length and height.</desc>
+    <rect width="50" height="50" fill="#cc0000" />
+</svg>
+```
+
+##### Further reading:
+* ["Using ARIA to enhance SVG accessibility"](https://developer.paciellogroup.com/blog/2013/12/using-aria-enhance-svg-accessibility/) - The Paciello Group
+* ["Accessible SVG Icons with Inline Sprites"](https://www.24a11y.com/2018/accessible-svg-icons-with-inline-sprites/) 24 Accessibility
+* ["Accessible SVG test page"](https://weboverhauls.github.io/demos/svg/)
+
+### Optimization
+
+It is quite common for SVG to make their way in to a project without being optimized. Many tools for creating SVG are notorious for including unnecessary markup. We recommend running all SVG through [SVGO(MG)](https://jakearchibald.github.io/svgomg/) or using tooling, like [gulp-svgmin](https://github.com/ben-eb/gulp-svgmin)
```

Comment:
@colorful-tones  Remove, "It is quite common for SVG to make their way in to a project without being optimized." It makes it sound like we don't optimize our SVGs